workers/unity/Packages/com.improbable.gdk.tools/Common.cs

```diff
@@ -66,65 +66,32 @@ internal static string GetThisPackagePath()
         /// </summary>
         public static string GetPackagePath(string packageName)
         {
-            var manifest = ParseDependencies(ManifestPath);
-
-            if (!manifest.TryGetValue(packageName, out var path))
+            // Get package info
+            var request = Client.List(true);
+            while (!request.IsCompleted)
```

Comment:
Yikes, what a terrible API. Busy-waiting the main thread...

Also, use a named parameter so people can tell what `true` means without reading the docs.

Reply:
It needs to be blocking, and sadly does not allow `await`, its pretty much instantly complete in offline mode anyways.

Will do the named variables on all calls there